### PR TITLE
Fix/bimdata menu

### PR DESCRIPTION
**Pull request by LrxGaelle**
- [x] I have tested my component
- [x] I have updated the documentation or there is no documentation needed

### CODE REVIEW

**Review by Kurtil — APPROVED** [review]

File: src/web/views/Components/Menu/props-basic-menu.js [hunk 1]
```diff
@@ -33,4 +33,10 @@ export default [
     "true",
     "If false, the menu component does not close when you click outside.",
   ],
+  [
+    "childrenLeft",
+    "Boolean",
+    "false",
+    "If true, the child menu will open to the left of the menu.",
```
Comment (Kurtil) [inline src/web/views/Components/Menu/props-basic-menu.js:40]: Present instead of future ?

Comment (Kurtil) [reply src/web/views/Components/Menu/props-basic-menu.js:40]: "the child menu opens to the left of the menu"

Comment (LrxGaelle) [reply src/web/views/Components/Menu/props-basic-menu.js:40]: Already present, but not added to the documentation

**Pull request merged**